src/globus_cli/commands/flows/start.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import annotations
+
+import typing as t
+import uuid
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import JSONStringOrFile, command
+from globus_cli.termio import Field, TextMode, display, formatters
+
+ROLE_TYPES = ("flow_viewer", "flow_starter", "flow_administrator", "flow_owner")
+
+
+@command("start", short_help="Start a flow")
+@click.argument(
+    "flow_id",
+    type=uuid.UUID,
+    metavar="FLOW_ID",
+)
+@click.option(
+    "--input",
+    "input_document",
+    type=JSONStringOrFile(),
+    help="""
+        The JSON input parameters used to start the flow.
+
+        The input document may be specified inline,
+        or it may be a path to a JSON file, prefixed with "file:".
+
+        Example: Inline JSON:
+
+        \b
+            --input '{"src": "~/source"}'
+
+        Example: Path to JSON file:
+
+        \b
+            --input file:parameters.json
+
+        If unspecified, the default is an empty JSON object ('{}').
+    """,
+)
+@click.option(
+    "--label",
+    type=str,
+    help="A label to give the run.",
+)
```

Comment:
The Flows service [doesn't require a label for the run](https://github.com/globusonline/flows/blob/main/globus/flows/views.py#L696-L703), so I didn't enforce anything here.

Reply:
I meant the code usage below made it look like the execution code would error out if `label` wasn't supplied. What happens if you start a flow with this command and don't supply a label?

Reply:
> What happens if you [...] don't supply a label?

The label will be set to `None`. When displaying the run, the web app will show "[Unlabeled Run 1234]", where "1234" is a large integer. When displaying the run label in a table of information, the web app will show an empty value.

Reply:
Thanks both for your input! I've updated the Python type annotation so that `label` is obviously nullable.

Note that the click type is still just `str`, as click expects a type that it can instantiate.